docs/release-notes/2.16.0.md

```diff
@@ -34,7 +34,12 @@ Percona Monitoring and Management (PMM) is a free and open-source platform for m
 
     Reworked the PMM Settings page to make it clear what features are in Technical Preview vs General Availability (GA) and to simplify activation/deactivation of technical preview features. We also provide a better definition of what a [Technical Preview](../details/glossary.md#technical-preview) is.
 
-* **Make non-dashboard pages part of grafana fork codebase (migration)**
+* **Migration of Settings and other service pages in PMM from Grafana dashboards**
+
+PMM Settings mage and others like Add instance or Inventory will be converted to grafana pages and will not be longer presented as a dashboard. 
```

Comment:
PMM Settings page and several others (including "Add Instance" and "Inventory" are being converted to Grafana pages and no longer presented as dashboards.  Additionally we're moving the menu to the sidebar navigation for consistency and more flexibility vs the older menu structure.